test_scripts/test_setups.py

```diff
@@ -105,8 +105,11 @@ class TestPhase(Enum):
 
 develop_setups = {'shane_kast_blue': ['452_3306_d57', '600_4310_d55', '830_3460_d46'],
                   'shane_kast_red': ['600_7500_d55_ret', '600_7500_d57', '600_5000_d46', '1200_5000_d57'],
+                  'gemini_gnirs': ['32_SB_SXD', '10_LB_SXD'],
+                  'gemini_gmos': ['GS_HAM_R400_700', 'GS_HAM_R400_860', 'GN_HAM_R400_885'],
+                  'gemini_flamingos2': ['HK_HK', 'JH_JH'],
                   'keck_deimos': ['600ZD_M_6500', '600ZD_tilted', '1200G_M_7750', '830G_LVM_8400', '830G_M_8100_26',
-                                  '830G_M_8500', '830G_L_8100'],
+                                  '830G_M_8500', '830G_L_8100', '1200B_Percy', '1200G_Ghassem'],
```

Comment:
Can we change the naming convention here to abide by the convention we already established?

Reply:
like what?

Reply:
Grating name, decker ('M' for multisit), central wavelength.

Reply:
done